crates/orderbook/src/arguments.rs

```diff
@@ -120,6 +120,10 @@ pub struct Arguments {
     /// to get them errors and our liveness check fails.
     #[clap(long, default_value = "24")]
     pub solvable_orders_max_update_age_blocks: u64,
+
+    /// Enable limit orders. Once the full limit order flow is implemented, this can be removed.
```

Comment:
nit: `clap` generates the `--help` text of the CLI argument from this comment. Would be weird to have this meta information in the help text (although it ultimately doesn't really matter).

Reply:
I'd rather keep it, I don't think it hurts :)